src/python_testing/TC_AVSM_2_1.py

```diff
@@ -292,145 +315,174 @@ async def test_TC_AVSM_2_1(self):
+            asserts.assert_is_not_none(value, "MicrophoneAGCEnabled is None")
 
         self.step(36)
         if self.pics_guard(self.check_pics("AVSM.S.A0022")):
             value = await self.read_single_attribute_check_success(endpoint=endpoint, cluster=cluster, attribute=attr.ImageRotation)
             logger.info(f"Rx'd ImageRotation: {value}")
+            asserts.assert_is_not_none(value, "ImageRotation is None")
+            matter_asserts.assert_valid_uint16(value, "ImageRotation")
 
         self.step(37)
         if self.pics_guard(self.check_pics("AVSM.S.A0023")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.ImageFlipHorizontal
             )
             logger.info(f"Rx'd ImageFlipHorizontal: {value}")
+            asserts.assert_is_not_none(value, "ImageFlipHorizontal is None")
 
         self.step(38)
         if self.pics_guard(self.check_pics("AVSM.S.A0024")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.ImageFlipVertical
             )
             logger.info(f"Rx'd ImageFlipVertical: {value}")
+            asserts.assert_is_not_none(value, "ImageFlipVertical is None")
 
         self.step(39)
         if self.pics_guard(self.check_pics("AVSM.S.A0025")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.LocalVideoRecordingEnabled
             )
             logger.info(f"Rx'd LocalVideoRecordingEnabled: {value}")
+            asserts.assert_is_not_none(value, "LocalVideoRecordingEnabled is None")
 
         self.step(40)
         if self.pics_guard(self.check_pics("AVSM.S.A0026")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.LocalSnapshotRecordingEnabled
             )
             logger.info(f"Rx'd LocalSnapshotRecordingEnabled: {value}")
+            asserts.assert_is_not_none(value, "LocalSnapshotRecordingEnabled is None")
 
         self.step(41)
         if self.pics_guard(self.check_pics("AVSM.S.A0027")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.StatusLightEnabled
             )
             logger.info(f"Rx'd StatusLightEnabled: {value}")
+            asserts.assert_is_not_none(value, "StatusLightEnabled is None")
 
         self.step(42)
         if self.pics_guard(self.check_pics("AVSM.S.A0028")):
             value = await self.read_single_attribute_check_success(
                 endpoint=endpoint, cluster=cluster, attribute=attr.StatusLightBrightness
             )
             logger.info(f"Rx'd StatusLightBrightness: {value}")
+            asserts.assert_is_not_none(value, "StatusLightBrightness is None")
```

Comment:
This also needs to test that the value is valid for a ThreeLevelAutoEnum

Reply:
Thank you for review. I had submitted it.